src/main/java/com/redhat/devtools/lsp4ij/launching/templates/LanguageServerTemplate.java

```diff
@@ -40,6 +40,7 @@ public String getName() {
     public static final String TEMPLATE_FILE_NAME = "template.json";
     public static final String INITIALIZATION_OPTIONS_FILE_NAME = "initializationOptions.json";
     public static final String SETTINGS_FILE_NAME = "settings.json";
+    public static final String CLIENT_SETTINGS_FILE_NAME = "clientSettings.json";

```

Comment:
Following the existing pattern of the server-side config file being named `settings.json` and naming the client-side config `clientSettings.json`.